<?xml version="1.0"?>
<launch>

    <arg name="des_x" default="10"/>
    <arg name="des_y" default="10"/>
    <arg name="des_theta"  default="1.9"/>

    <param name="des_pos_x" value="$(arg des_x)" />
    <param name="des_pos_y" value="$(arg des_y)" />
    <param name="des_pos_yaw" value="$(arg des_theta)" />


  <!-- Load the URDF into ROS parameter server -->
  <param name="robot_description" command="$(find xacro)/xacro '$(find summit_description)/robot/summit.urdf.xacro' --inorder"/>

  <!-- Publish joint values -->
  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" >
    <remap from="/joint_states" to="/summit/joint_states" />
  </node>

  <!-- convert joint states to TF transforms for rviz, etc  -->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher">
    <remap from="/joint_states" to="/summit/joint_states" />
  </node>

  <!-- Spawn model -->
  <node name="urdf_spawner" pkg="gazebo_ros" type="spawn_model" respawn="false" output="screen"
    args="-urdf -x 0 -y 0 -z 0  -model summit1 -param robot_description"/>

  <node pkg="summit_description" type="goToPoint_omni.py" name="go_to_point" output="screen" />

 </launch>